### ISSUE TITLE
Disallow deletion of DC datastream

### ISSUE THREAD

**Issue by mjordan** [SFULibrary/islandora_datastream_crud#23]
Following up on #21, we should not allow users to delete the DC datastream.:

* https://wiki.duraspace.org/display/FEDORA38/Fedora+Digital+Object+Model says "Every Fedora digital object has one "DC" (Dublin Core) Datastream by default which is used to contain metadata about the object (and will be created automatically if one is not provided)."
* the Fedora admin client throws an error when you try to purge the DC datastream
* the Islandora datastream admin UI doesn't provide a 'delete' action.

As per @bondjimbond's suggestion, Datastream CRUD should provide a useful error message to the user.

**Comment by mjordan** [comment]
@bondjimbond, care to test the issue-23 branch? If you specify DC as the datastream to delete, you get the message "DC is a required datastream and cannot be deleted." and drush exits.

```
drush islandora_datastream_crud_delete_datastreams --user=admin --dsid=DC --pid_file=/tmp/issue23.pids
```

**Comment by bondjimbond** [comment]
Looks good to me!

**Comment by mjordan** [comment]
Thanks, merged with 4a250e96bc0afcc0563263037d5e621a6fe7c217.
